pygaggle/rerank/transformer.py

```diff
@@ -27,19 +27,27 @@
 
 class MonoT5(Reranker):
     def __init__(self,
-                 model_name_or_instance: Union[str, T5ForConditionalGeneration] = 'castorini/monoT5-base-msmarco',
-                 tokenizer_name_or_instance: Union[str, QueryDocumentBatchTokenizer] = 't5-base'):
-        if isinstance(model_name_or_instance, str):
-            device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
-            model_name_or_instance = T5ForConditionalGeneration.from_pretrained(model_name_or_instance).to(device).eval()
-        self.model = model_name_or_instance
-
-        if isinstance(tokenizer_name_or_instance, str):
-            tokenizer_name_or_instance = T5BatchTokenizer(AutoTokenizer.from_pretrained(tokenizer_name_or_instance), batch_size=8)
-        self.tokenizer = tokenizer_name_or_instance
-
+                 model: T5ForConditionalGeneration = None,
+                 tokenizer: QueryDocumentBatchTokenizer = None):
+        self.model = model or self.get_model()
+        self.tokenizer = tokenizer or self.get_tokenizer()
```

Comment:
I'm not sure but is such code standard python practice?

Reply:
@ronakice, is your concern about the use of  the `or` operand?  I've seen this being used as a shorthand to
```python
self.model = model if model else self.get_model()
```
so I don't think there's any issues there. Or is it more about the PR in general?